chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/ChatManagerFixtures.kt

```diff
@@ -55,26 +54,6 @@ class TestChatkitDependencies(
     override val appHooks = NoAppHooks()
 }
 
-val SynchronousCurrentUser.generalRoom
-    get() = rooms.find { it.name == Rooms.GENERAL } ?: error("Could not find room general")
-
-private val managers = ConcurrentLinkedQueue<SynchronousChatManager>()
-
-fun chatFor(userName: String) = SynchronousChatManager(
```

Comment:
This file now can be renamed as it has nothing to do with ChatManager that is gone. It also no longer represents fixture, i.e. setting up a test. Maybe naming it simply as `TestUtil.kt` for now? Later we can think about splitting it into more specific files.

Reply:
Good point - have updated!

Reply:
After failing the build, it turned out we already had a `TestUtil` file - I renamed this one to `ChatManagerTestUtil.kt` :/

Reply:
Oh yes, sorry it was even me recently creating `TestUtil.kt`. Well at least I scored on consistency! ;) Then lest rename this file to `TestChatkitDependencies.kt` and also let move its `assumeSuccess` to `TestUtil.kt`.

Reply:
👍 have updated both those pieces.